core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -208,7 +208,12 @@ private void populateValues(HttpServletRequest request) {
     }
 
     String ufsRoot = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS);
-    UnderFileSystem ufs = UnderFileSystem.Factory.createForRoot();
+    UnderFileSystem ufs = null;
+    try {
+      ufs = UnderFileSystem.Factory.createForRoot();
+    } catch (Exception e) {
+      LOG.error("Failed to create root UFS {}: {}", ufsRoot, e.getMessage());
```

Comment:
we'll end up with NPEs below and still stack trace the UI, maybe we could isolate this to a `setUfsAttributes` method, and just return early if we can't get the UFS

Reply:
Done.